gpt_playground/settings.py

```diff
@@ -548,6 +548,7 @@
     "account_reset_password_from_key",
     "teams:signup_after_invite",
     "account_login",
+    "teams:delete_team",
```

Comment:
doesn't fix the "No team found for audit context" errors since the request is `None` [here](https://github.com/dimagi/open-chat-studio/blob/main/apps/audit/auditors.py#L22) and then by [this check](https://github.com/dimagi/open-chat-studio/blob/main/apps/audit/auditors.py#L81) it does not return a view name